server/server.js

```diff
@@ -65,6 +68,11 @@ app.post('/command', (req, res) => {
   }
 });
 
+app.post('/changeDir', (req, res) => {
```

Comment:
this function will help us in development using Postman to change the working directory

Reply:
this comment should be inline in the code. 
moreover, please change the route name to something like `DEVchangeDir` or something that indicates that it's for development purposes